examples/resources/powerstore_volume/volume.tf

```diff
@@ -7,7 +7,7 @@
 
 resource "powerstore_volume" "test1" {
   name = "test_vol1"
-  size = 7516192768
+  size = 8
```

Comment:
Please refer to the example present in main.tf, capacity_unit should also be added along with size.